tests/baselines/reference/assignmentLHSIsValue.errors.txt

```diff
@@ -109,6 +110,8 @@ tests/cases/conformance/expressions/assignmentOperator/assignmentLHSIsValue.ts(7
     
     // array literals
     ['', ''] = value;
+    ~~~~~~~~
+!!! error TS2461: Type 'any' is not an array type.

```

Comment:
we already have two others errors after that:
!!! error TS2364: Invalid left-hand side of assignment expression.
 !!! error TS2364: Invalid left-hand side of assignment expression.